services/app-web/src/pages/StateSubmission/Contacts/Contacts.tsx

```diff
@@ -129,6 +129,38 @@ const actuaryContactErrorHandling = (
     return error
 }
 
+// Convert the formik errors into a shape that can be passed to ErrorSummary
+const flattenErrors = (
+    errors: FormikErrors<ContactsFormValues>
+): { [field: string]: string } => {
+
+    const flattened: {[field: string]: string} = {}
+
+    if (errors.stateContacts && Array.isArray(errors.stateContacts)) {
```

Comment:
There might be some clever way tot use [Array.flatMap()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) here but I am a big fan of this simplicity.

Reply:
This code is right on the line where if it gets more complicated (or repeats itself a third time) it might be worth some additional refinement. I'm happy to do that sooner if we think it's worthwhile.

Reply:
nah it’s fine